modules/canbus/vehicle/vehicle_controller.h

```diff
@@ -152,6 +153,7 @@ class VehicleController {
 
  protected:
   canbus::VehicleParameter params_;
```

Comment:
delete params_?

Reply:
@startcode It is still half-done,  so I left the legacy here for now